apps/mobile/src/screens/chat/libs/components/loader/libs/enums/delay.enum.ts

```diff
@@ -0,0 +1,7 @@
+const DotDelays = {
```

Comment:
```suggestion
const DotDelay = {
```
Change file name to `dot-delay.enum.ts`